tests/test_jats.py

```diff
@@ -20,6 +20,7 @@
 
 
 class TestJATS(unittest.TestCase):
+    maxDiff = None
```

Comment:
Remove this after you're done with testing.

Reply:
Removed `maxDiff = None` line.